.changeset/ten-peaches-tickle.md

```diff
@@ -0,0 +1,14 @@
+---
+"@builder.io/sdk-react-native": patch
+"@builder.io/sdk-angular": patch
+"@builder.io/react": patch
+"@builder.io/sdk": patch
+"@builder.io/sdk-react-nextjs": patch
+"@builder.io/sdk-qwik": patch
+"@builder.io/sdk-react": patch
+"@builder.io/sdk-solid": patch
+"@builder.io/sdk-svelte": patch
+"@builder.io/sdk-vue": patch
+---
+
+feat: add additional breakpoint xsmall support for gen-2 sdks
```

Comment:
Since this will be in the release notes of individual SDKs, it doesn't make sense to specify "gen-2 SDKs":

```suggestion
feat: Add support for `xsmall` additional breakpoint.
```